pkg/engine/engine_test.go

```diff
@@ -1188,3 +1188,82 @@ func TestEngineInitializesCloudProviderDetectors(t *testing.T) {
 		t.Fatal("no detectors found implementing Endpoints(), did EndpointSetter change?")
 	}
 }
+
+func TestEngineignoreLine(t *testing.T) {
```

Comment:
This test suite may not be the most comprehensive—particularly since the issue stemmed from a bug in the Postgres detector—but alas, it’s still valuable. 😅

Reply:
Starting a bug fix with a failing test is a great way to work.

I would only remove this test if you can remove your fix and another test in the Postgres detector will pick it up.